kratos/includes/variables.h

```diff
@@ -124,6 +124,8 @@ namespace Kratos
 
     KRATOS_DEFINE_VARIABLE( double, GRADIENT_PENALTY_COEFFICIENT )
 
+    KRATOS_DEFINE_VARIABLE( double, THETA )
```

Comment:
I'd find a more descriptive name. What comes to my mind with `THETA` is an angle. Maybe `CRANK_NICOLSON_THETA` following what is done below for the Newmark scheme,

Reply:
I used THETA because that's what is being used in ConvectionDiffussionApp. Probably CRANK_NICOLSON_THETA  is the most understandable, though it is not 100% technically correct.

Reply:
Agree... It is only Crank-Nicolson theta when theta is 0.5... What about `TIME_INTEGRATION_THETA`?

Reply:
Agree!